### PR TITLE
Release Management Notes

### PR DESCRIPTION
**Pull request by sneridagh**
Definition of breaking

### CODE REVIEW

File: docs/source/release-management-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+The private API is everything that is considered "internal" to Volto and it is intended only to be used by Volto itself.
+
+The following Volto components are considered Private API:
+
+- Libraries used by Volto
+
+### Example
+
+If a bug is found in the text block and the solution involves to upgrade the Slate library, it won't be considered a breaking change.
+Even if that upgrade implies a breaking change on itself.
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:59]:
```suggestion
This is true, even if that upgrade implies a breaking change in itself.

```

File: docs/source/release-management-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+myst:
+  html_meta:
+    "description": "Release Process Information"
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:4]:
```suggestion
    "description": "Release management notes for Volto"
```

File: docs/source/release-management-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "property=og:description": "Volto Release Notes for the Plone content management system"
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:5]:
```suggestion
    "property=og:description": "Release management notes for Volto"
```

File: docs/source/release-management-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    "property=og:title": "Volto Release Notes"
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:6]:
```suggestion
    "property=og:title": "Volto release management notes"
```

File: docs/source/release-management-notes/index.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    "keywords": "Volto, Plone, frontend, Release Notes, change log, changelog, change history"
+---
+
+# Release Management Notes
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:10]:
```suggestion
# Release management notes
```

File: docs/source/release-management-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Definition of Breaking
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:12]:
```suggestion
## Definition of breaking
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Some spelling, grammar, syntax, and style guide improvements.

**Review by stevepiercy — APPROVED** [review]
LGTM. Once merged, I would add reference links to it from the PRs:

- https://github.com/plone/volto/pull/5385
- https://github.com/plone/volto/pull/5404

**Comment by stevepiercy** [comment]
This file needs to be reconciled against the file `RELEASING.md` in the root of the Volto repo or monorepo before merging. See also https://github.com/plone/volto/pull/5409

**Comment by sneridagh** [comment]
@tisto @mauritsvanrees It's been a while I've started to write this. What do you think? We can merge this, and continue iterating over it, but I think it's good enough for now.

File: docs/source/release-management-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+The following Volto components are considered Private API:
+
+- CMS user interface fundamental components (`./src/components/manage`)
+- CMS user interface control panels
+- Libraries used by Volto
+
+```{note}
+In general, the CMSUI is not considered public API, because the customization/modification in a project is not usual, and the overall experience is tied to these components.
+So we prefer give preference to maintain a good user experience over not backporting changes that could be breaking to the product but could be beneficial for the vast majority of use cases.
```
Comment (mauritsvanrees) [inline docs/source/release-management-notes/index.md:68]: "prefer give preference" is double.  I don't mind how to fix this, but this is one way:

```suggestion
So we give preference to maintain a good user experience over not backporting changes that could be breaking to the product but could be beneficial for the vast majority of use cases.
```

**Review by mauritsvanrees — APPROVED** [review]
Makes sense to me.

One small suggestion on improving the wording.

**Comment by stevepiercy** [comment]
Should there be references to the following, explaining how all these documents complement one another?

- https://github.com/plone/volto/blob/main/RELEASING.md
- https://plone.org/download/release-schedule

**Comment by sneridagh** [comment]
@stevepiercy I don't think that the first one needs to reference here.

I guess that it makes sense to have it in: https://plone.org/download/release-schedule

@mauritsvanrees what do you think?

**Comment by mauritsvanrees** [comment]
> @stevepiercy I don't think that the first one needs to reference here.

Agreed.

> I guess that it makes sense to have it in: https://plone.org/download/release-schedule

Adding a link to the release schedule/policy would indeed be good.  Probably in the paragraph directly under the main 'Release management notes'

File: docs/source/release-management-notes/index.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    "description": "Release management notes for Volto"
+    "property=og:description": "Release management notes for Volto"
+    "property=og:title": "Volto release management notes"
+    "keywords": "Volto, Plone, frontend, Release Notes, change log, changelog, change history"
+---
+
+# Release management notes
+
+This chapter describes how maintainers manage the releases of Volto.
+
```
Comment (stevepiercy) [inline docs/source/release-management-notes/index.md:13]:
````suggestion

```{seealso}
Read the [Plone Release Schedule](https://plone.org/download/release-schedule) for details of how Volto's release management works in Plone.
```


````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Here's a suggested phrase and admonition for the link.

**Review by stevepiercy — APPROVED** [review]
I pushed a commit with some grammar fixes, MyST enhancements, and cross-reference linking.

File: docs/source/release-management-notes/index.md [hunk 9]
```diff
@@ -52,35 +50,37 @@ This especially includes `className` props that would cause existing CSS to fail
...
 
-If there is a change in the CMS user interface that implies a change of behavior and/or change of markup that improves the overall user experience, it's not considered a breaking change.
-It will be properly expressed and notified in the changelog notes of a minor change.
+## Examples of breaking and non-breaking changes
+
+If there is a change in the CMS user interface that implies either a change of behavior or change of markup that improves the overall user experience, it's not a breaking change.
+It will be properly identified in the {doc}`../release-notes/index` as a minor change.
 
-Any change to a control panel.
+Any change to a control panel is a breaking change.
```
[the remark this answers is not in the document]
Comment (stevepiercy) [reply docs/source/release-management-notes/index.md]: Addressed in https://github.com/plone/volto/pull/5358/commits/2fcdcc90fe1689574c2e60aada23c8fbd7d2f03e

**Pull request merged**